contracts/liquidity_hub/pool-manager/src/queries.rs

```diff
@@ -20,11 +21,21 @@ use crate::{math::Decimal256Helper, state::SWAP_ROUTES};
 /// Query the native token decimals
 pub fn query_native_token_decimal(
     deps: Deps,
+    pair_identifier: String,
     denom: String,
```

Comment:
do we need both, `pair_identifier` and `denom` here again?

Reply:
we need them to query, but perhaps not on the response